docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -0,0 +1,301 @@
+include::variables.adoc[]
+
+[[setup-virtualization-environment]]
+= Setting Up the Virtualization Environment
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:next-steps: Proceed to xref:../getting-started/installing.adoc#[Installing {project}] once your hypervisor has been installed and configured.
+
+toc::[]
+
+[[setup-virtualization-environment-overview]]
+== Overview
+
+Follow the appropriate procedure to set up virtualization for your particular operating system and hypervisor.
+{project} uses Docker Machine and its driver plug-in architecture to provide a consistent way to manage the OpenShift VM.
```

Comment:
s/OpenShift VM/ Minishift VM/

Reply:
Actually we use https://github.com/docker/machine/tree/master/libmachine , which is the library from the Docker Machine code base.  I guess we do not need to be more accurate in the description, do we?

Reply:
Personally, I don't think that it adds anything to the documentation to be that specific. I think "Docker Machine" conveys the idea perfectly well.

On that note, though, it may be worth rewording as "the Docker Machine libmachine library" or something along similar lines.